### ISSUE TITLE
[Roll20] Update Schedule

### ISSUE THREAD

**Issue by Stexinator** [Roll20/roll20-character-sheets#8709]
Is it possible to get information when the next updates will be merged/go live?

**Comment by vince-roll20** [comment]
Would love to see an alert or a banner on the pulls page showing the next updates. ;-)

**Comment by Noisyflasher** [comment]
I was sure that this information available in the documentation: https://wiki.roll20.net/Beginner%27s_Guide_to_GitHub

**Comment by vince-roll20** [comment]
Maybe it's something that we can bring up at the next roundtable discussion...

**Comment by Stexinator** [comment]
> 
> 
> I was sure that this information available in the documentation: https://wiki.roll20.net/Beginner%27s_Guide_to_GitHub

I could not find information there.

None the less I was told from Roll20 Staff that merges always happen on Tuesday. But this does not hold true, as can be seen this week.

Another point is that the hotfix workflow does not work. The only way I see to have a hotfix happen is if there is enough of a stink in the forums. And I think this is a bad workflow

**Comment by Noisyflasher** [comment]
> I could not find information there.
![image](https://user-images.githubusercontent.com/5252033/119781305-eab86380-bedb-11eb-92b5-894ac78ac641.png)

> 
> Another point is that the hotfix workflow does not work. The only way I see to have a hotfix happen is if there is enough of a stink in the forums. And I think this is a bad workflow
Totally agree. I pushed a new character sheet yesterday which looks good in the sandbox but after release I found a problems with layout in production. I need a small styles-names fix but now I need to wait for a week before the update.

**Comment by nmbradley** [comment]
Hi Folks,

In response to this request, we have increased our frequency of merging sheet changes over the last two weeks. We will continue this practice going forwards. The trade off for this increased frequency is less consistency. We will continue to treat Monday as 'the' day for making these merges, but will make other merges during the week, as time permits. 

Closing this issue, but happy to hear further comments or observations as relevant.
